src/client.ts

```diff
@@ -174,6 +174,20 @@ export class Client {
     return this._packStateData();
   }
 
+  /**
+   * `getFwVersion` gets the firmware version of the paired device.
+   * @returns Either an object with semver properties (fix, minor, and major) or undefined.
+   */
+  public getFwVersion (): { fix: number, minor: number, major: number } | undefined {
```

Comment:
Do you prefer `undefined`? I usually use `null` for stuff like this. Either obviously works

Reply:
No, I prefer `null`. It was already returning `undefined` because the function didn't explicitly have another return value, but yeah I like `null` better. It actually explains what's going on.

`b4d28572dc68b6660f32df808a86998660f9834a`